packages/react-components/react-nav-preview/src/components/NavItem/useNavItemStyles.styles.ts

```diff
@@ -23,19 +37,44 @@ const useContentStyles = makeStyles({
   selected: typographyStyles.body1Strong,
 });
 
+const indicatorHeight = '20px';
+
+const useIndicatorStyles = makeStyles({
+  base: {
+    '::after': {
+      position: 'absolute',
+      transform: 'translateX(-18px)', // per spec
+      backgroundColor: tokens.colorNeutralForeground2BrandSelected,
+      width: '4px', // No relevant to any design token for these
```

Comment:
I would recommend adding these to a `navItemVars` object, something like:

```ts
const navItemTokens = {
  indicatorWidth: '4px',
  indicatorOffset: '18px'
}
```

```suggestion
      transform: `translateX(-${navItemTokens.indicatorOffset})`, // per spec
      backgroundColor: tokens.colorNeutralForeground2BrandSelected,
      width: navItemTokens.indicatorWidth, // No relevant to any design token for these
```

Reply:
Yeah - I like this a lot - I'm going to be reusing a lot of these constants throughout the component set.